client/src/components/bpc.js

```diff
@@ -6,20 +6,30 @@ const request = (path, options = {}) => {
 
   if(typeof local_bpc_url === 'string' && local_bpc_url.length > 0) {
     bpc_url = local_bpc_url;
+  } else if (process.env.BPC_URL !== '') {
+    bpc_url = process.env.BPC_URL;
   }
+  console.log('BPC URL:', bpc_url);
 
   if(!bpc_url) {
     console.error('BPC URL missing');
     return;
   }
 
-  const _request = new Request(`${ bpc_url }${ path }`, options);
+  const opt = {
+      ...options,
+      headers: {
+        ...options.headers,
+        'content-Type': 'application/json'
```

Comment:
all letters lowercased or first capitals
```suggestion
        'Content-Type': 'application/json'
```

Reply:
Also, are you sure that we want to add this content type for all sort of requests? I would assume that yes, but all bpc requests from console should be then listed and tested by QA then

Reply:
lets try this, @bmxsyka will test it